depot/.gitignore

```diff
@@ -21,3 +21,10 @@
 .byebug_history
 /assets/images/*
 /.fuse*
+
+# Configuration files
+
+config/app_config.yml
```

Comment:
config/*.yml

Reply:
also add database.yml.example and secrets.yml.example. discuss if you have any doubts.